<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TinyGTestpanel</class>
 <widget class="QDialog" name="TinyGTestpanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TinyG Testpanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="lstMessages"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtCommand"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuery">
       <property name="text">
        <string>Query</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
